mlrun/api/api/utils.py

```diff
@@ -123,12 +125,26 @@ def _parse_submit_job_body(db_session: Session, data):
     return function, task
 
 
-def submit(db_session: Session, data):
+async def submit_task(db_session: Session, data):
```

Comment:
`_parse_submit_run_body` was just a left over from the first renaming (job -> run, after which I thought that task is more appropriate)...
as I said in my first comment, to me it sounds like you're submitting a Task (to be executed by the function) and the output is a Run object, you're not submitting a Run. (i.e. since the verb is submit, it feels like the noun should be what you give not what you get)
here is exactly the place where MLRun "gets a task" - the data here has a task.
Looking on the docs of MLRun's basic components - https://github.com/mlrun/mlrun/#basic-components:
> The run object is created as a result of running a task on a function
(IMO the verb should have been submitting instead running)
WDYT ?

Reply:
Again - in plain english I would agree, and it makes sense considering that sentence, except the sentence makes no sense since the docs make a mishmash of what things are and says "running" a task.
But as for the mlrun definition - I think this is slightly inaccurate. It only makes sense because input is improperly named "task".
So it makes "english" sense (you submit a task, but you say that because a task is a long running process that submitting it means launching it). But, imagine that we rename "task" to what it is - "run_input" or "inputs"
now it's not reflecting, I think, what you want to reflect - "submit_run_inputs". I *think* the naming here tries to reflect that the "run" is launched, not that the "inputs" are ingested.

Reply:
changed to run